DEV_GUIDE.md

```diff
@@ -278,3 +278,22 @@ Finally, before committing your changes you'll want to clean up
 ```sh
 rm foo.txt .fossa.yml
 ```
+
+# MAC Developer Notes
+
+It's critically important to ensure that you have the correct JDK v8 installed before you can successfully compile the Stargate project. There are a number of versions of JDK 8 and a number of different ways to install them, but not all of them will work successfully with Stargate.
+
+Download and install Maven from this link: https://maven.apache.org/download.cgi
```

Comment:
We include the maven wrapper so this step isn't required.

Reply:
Ok. Sounds good.